src/components/categoryButton.tsx

```
@@ -0,0 +1,41 @@
+            : `/?category_id=${category.category_id}`;
+
+        return (
+          <Link
+            href={href}
+            scroll={false}
+            key={category.category_id}
+            className={
+              category.category_id === currentCategoryId ? "active" : ""
+            }
+          >
+            <button>{category.name}</button>
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)